search_action_domain/migrations/15.0.1.0.0/post-migration.py

```diff
@@ -0,0 +1,26 @@
+import logging
+
+_logger = logging.getLogger(__name__)
+
+
+def migrate(cr, version):
+    convert_export_reference_to_search_action_domain_for_filter(cr)
+
+
+def convert_export_reference_to_search_action_domain_for_filter(cr):
+    cr.execute(
+        """
+        UPDATE
+            ir_model_data
+        SET
+            module = 'search_action_domain'
```

Comment:
Missing comma, syntax error:

```suggestion
            module = 'search_action_domain',
```

Reply:
Done!